### PR TITLE
[MRG] TST add (simplified and normal) image classifier test kit

### PR DESCRIPTION
**Pull request by glemaitre**
Adding a simple kit based on digits from scikit-learn for image classification.

### CODE REVIEW

**Comment by kegl** [comment]
There is also an `image_classifier` workflow which has a separate `image_preprocessor` workflow element, maybe you can add this also to the PR. You can use the same data.

File: rampwf/tests/test_kits.py [hunk 1]
```diff
@@ -10,9 +11,29 @@
 PATH = os.path.dirname(__file__)
 
 
+def skip_windows_py27():
+    return pytest.mark.skipif(
+        (sys.platform == 'win32') and (sys.version_info < (3, 5)),
+        reason="tensorflow not available")
```
Comment (jorisvandenbossche) [inline rampwf/tests/test_kits.py]: This one is not needed any more?

File: rampwf/tests/test_kits.py [hunk 2]
```diff
@@ -10,9 +11,29 @@
...
-    return [(os.path.abspath(path_kit,))
-            for path_kit in sorted(glob.glob(os.path.join(PATH, 'kits', '*')))]
+    grid = []
+    for path_kit in sorted(glob.glob(os.path.join(PATH, 'kits', '*'))):
+        if 'digits' in path_kit:
+            grid.append(pytest.param(os.path.abspath(path_kit),
+                                     marks=skip_no_tensorflow()))
+        else:
+            grid.append(os.path.abspath(path_kit))
+    return grid
```
Comment (jorisvandenbossche) [inline rampwf/tests/test_kits.py]: I personally actually preferred the manual list, but I should have said that in a previous PR .. :)

**Comment by glemaitre** [comment]
@kegl @jorisvandenbossche @aboucaud I think this is ready to be merged

**Comment by glemaitre** [comment]
@kegl @agramfort Do you agree to merge this PR?

**Comment by agramfort** [comment]
thx @glemaitre and @jorisvandenbossche !

**Pull request merged**